pkg/json_raw_results.go

```diff
@@ -508,9 +508,9 @@ func (r *jsonScorecardRawResult) addFuzzingRawResults(fd *checker.FuzzingData) e
 			URL:  f.URL,
 			Desc: f.Desc,
 		}
-		if f.File != nil {
+		if f.File != nil && len(f.File) == 1 {
```

Comment:
why restrict to `len(f.File) == 1`? Let's return all the findings:.

Reply:
I notice in `type jsonTool`:

```
type jsonTool struct {
	URL  *string          `json:"url"`
	Desc *string          `json:"desc"`
	Job  *jsonWorkflowJob `json:"job,omitempty"`
	File *jsonFile        `json:"file,omitempty"`
	Name string           `json:"name"`
	// TODO: Runs, Issues, Merge requests.
}
```

the property `File` is still a type of pointer `*jsonFile`. Should I also modify this one to an array, like `[]jsonFile`?